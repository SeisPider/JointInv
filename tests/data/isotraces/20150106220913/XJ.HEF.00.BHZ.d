SURF96 R U X -1 90.00000 3.58251 0.22075
SURF96 R U X -1 85.00000 3.65814 0.21738
SURF96 R U X -1 80.00000 3.76522 0.21675
SURF96 R U X -1 75.00000 3.73295 0.19973
SURF96 R U X -1 70.00000 3.67379 0.18056
SURF96 R U X -1 65.00000 3.71850 0.17176
SURF96 R U X -1 60.00000 3.83313 0.16848
SURF96 R U X -1 55.00000 3.79714 0.15155
SURF96 R U X -1 50.00000 3.74161 0.13377
SURF96 R U X -1 48.00000 3.68805 0.12477
SURF96 R U X -1 46.00000 3.60830 0.11446
SURF96 R U X -1 44.00000 3.55497 0.10627
SURF96 R U X -1 42.00000 3.50708 0.09872
SURF96 R U X -1 40.00000 3.46819 0.09195
SURF96 R U X -1 38.00000 3.42132 0.08501
SURF96 R U X -1 36.00000 3.37932 0.07857
SURF96 R U X -1 34.00000 3.33590 0.07231
SURF96 R U X -1 32.00000 3.25425 0.06476
SURF96 R U X -1 29.00000 3.12646 0.05417
SURF96 R U X -1 28.00000 3.13288 0.05252
SURF96 R U X -1 28.00000 3.41999 0.06259
SURF96 R U X -1 28.00000 3.32690 0.05923
SURF96 R U X -1 27.00000 3.14612 0.05107
SURF96 R U X -1 27.00000 3.41603 0.06021
SURF96 R U X -1 26.00000 3.15081 0.04933
