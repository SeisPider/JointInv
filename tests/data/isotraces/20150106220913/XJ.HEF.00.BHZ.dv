MFT96 R U T -1 91.022    3.6065    1.0000    5232.6     0.3245E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 89.043    3.6232    1.0000    5232.6     0.3695E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 87.149    3.6390    1.0000    5232.6     0.4175E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 85.333    3.6540    1.0000    5232.6     0.4680E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 83.592    3.6683    1.0000    5232.6     0.5206E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 81.920    3.6817    1.0000    5232.6     0.5750E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 80.314    3.6944    1.0000    5232.6     0.6306E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 78.769    3.7062    1.0000    5232.6     0.6869E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 77.283    3.7172    1.0000    5232.6     0.7434E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 75.852    3.7274    1.0000    5232.6     0.7996E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 74.473    3.7368    1.0000    5232.6     0.8550E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 73.143    3.7454    1.0000    5232.6     0.9090E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 71.860    3.7532    1.0000    5232.6     0.9612E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 70.621    3.7603    1.0000    5232.6     0.1011E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 69.424    3.7665    1.0000    5232.6     0.1058E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 68.267    3.7719    1.0000    5232.6     0.1102E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 67.148    3.7766    1.0000    5232.6     0.1143E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 66.065    3.7805    1.0000    5232.6     0.1179E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 65.016    3.7837    1.0000    5232.6     0.1211E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 64.000    3.7861    1.0000    5232.6     0.1239E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 63.015    3.7878    1.0000    5232.6     0.1262E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 62.061    3.7888    1.0000    5232.6     0.1280E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 61.134    3.7891    1.0000    5232.6     0.1294E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 60.235    3.7887    1.0000    5232.6     0.1303E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 59.362    3.7877    1.0000    5232.6     0.1309E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 58.514    3.7860    1.0000    5232.6     0.1312E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 57.690    3.7837    1.0000    5232.6     0.1312E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 56.889    3.7807    1.0000    5232.6     0.1311E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 56.110    3.7772    1.0000    5232.6     0.1311E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 55.351    3.7731    1.0000    5232.6     0.1312E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 54.613    3.7685    1.0000    5232.6     0.1317E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 53.895    3.7633    1.0000    5232.6     0.1328E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 53.195    3.7576    1.0000    5232.6     0.1346E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 52.513    3.7514    1.0000    5232.6     0.1374E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 51.848    3.7447    1.0000    5232.6     0.1413E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 51.200    3.7376    1.0000    5232.6     0.1465E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 50.568    3.7301    1.0000    5232.6     0.1532E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 49.951    3.7221    1.0000    5232.6     0.1615E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 49.349    3.7138    1.0000    5232.6     0.1716E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 48.762    3.7051    1.0000    5232.6     0.1835E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 48.188    3.6960    1.0000    5232.6     0.1973E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 47.628    3.6866    1.0000    5232.6     0.2131E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 47.080    3.6769    1.0000    5232.6     0.2308E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 46.545    3.6669    1.0000    5232.6     0.2504E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 46.022    3.6567    1.0000    5232.6     0.2719E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 45.511    3.6462    1.0000    5232.6     0.2950E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 45.011    3.6354    1.0000    5232.6     0.3195E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 44.522    3.6245    1.0000    5232.6     0.3452E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 44.043    3.6134    1.0000    5232.6     0.3716E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 43.574    3.6021    1.0000    5232.6     0.3985E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 43.116    3.5906    1.0000    5232.6     0.4255E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 42.667    3.5790    1.0000    5232.6     0.4521E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 42.227    3.5673    1.0000    5232.6     0.4779E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 41.796    3.5555    1.0000    5232.6     0.5025E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 41.374    3.5436    1.0000    5232.6     0.5257E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 40.960    3.5317    1.0000    5232.6     0.5469E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 40.554    3.5196    1.0000    5232.6     0.5661E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 40.157    3.5076    1.0000    5232.6     0.5830E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 39.767    3.4955    1.0000    5232.6     0.5975E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 39.385    3.4834    1.0000    5232.6     0.6095E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 39.010    3.4714    1.0000    5232.6     0.6192E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 38.642    3.4593    1.0000    5232.6     0.6266E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 38.280    3.4473    1.0000    5232.6     0.6319E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 37.926    3.4353    1.0000    5232.6     0.6355E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 37.578    3.4234    1.0000    5232.6     0.6376E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 37.236    3.4116    1.0000    5232.6     0.6387E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 36.901    3.3998    1.0000    5232.6     0.6392E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 36.571    3.3882    1.0000    5232.6     0.6397E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 36.248    3.3766    1.0000    5232.6     0.6405E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 35.930    3.3652    1.0000    5232.6     0.6424E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 35.617    3.3539    1.0000    5232.6     0.6458E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 35.310    3.3428    1.0000    5232.6     0.6512E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 35.009    3.3318    1.0000    5232.6     0.6591E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 34.712    3.3209    1.0000    5232.6     0.6696E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 34.420    3.3103    1.0000    5232.6     0.6830E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 34.133    3.2998    1.0000    5232.6     0.6992E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 33.851    3.2895    1.0000    5232.6     0.7179E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 33.574    3.2794    1.0000    5232.6     0.7387E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 33.301    3.2695    1.0000    5232.6     0.7609E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 33.032    3.2599    1.0000    5232.6     0.7838E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.768    3.2504    1.0000    5232.6     0.8065E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.508    3.2412    1.0000    5232.6     0.8281E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.252    3.2322    1.0000    5232.6     0.8478E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.000    3.2235    1.0000    5232.6     0.8646E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 31.752    3.2151    1.0000    5232.6     0.8779E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 31.508    3.2069    1.0000    5232.6     0.8870E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 31.267    3.1989    1.0000    5232.6     0.8915E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 31.030    3.1913    1.0000    5232.6     0.8911E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.797    3.1839    1.0000    5232.6     0.8857E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.567    3.1769    1.0000    5232.6     0.8754E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.341    3.1701    1.0000    5232.6     0.8603E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.118    3.1637    1.0000    5232.6     0.8411E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.898    3.1575    1.0000    5232.6     0.8181E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.681    3.1517    1.0000    5232.6     0.7920E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.468    3.1463    1.0000    5232.6     0.7635E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.257    3.1411    1.0000    5232.6     0.7335E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.050    3.1363    1.0000    5232.6     0.7025E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.845    3.1319    1.0000    5232.6     0.6715E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.643    3.1278    1.0000    5232.6     0.6409E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.444    3.1241    1.0000    5232.6     0.6113E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.248    3.1208    1.0000    5232.6     0.5830E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.055    3.1179    1.0000    5232.6     0.5565E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.864    3.1154    1.0000    5232.6     0.5317E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.676    3.1132    1.0000    5232.6     0.5086E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.490    3.1115    1.0000    5232.6     0.4873E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.307    3.1102    1.0000    5232.6     0.4674E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.126    3.1094    1.0000    5232.6     0.4489E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.947    3.1090    1.0000    5232.6     0.4314E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.771    3.1090    1.0000    5232.6     0.4147E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.597    3.1095    1.0000    5232.6     0.3985E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.426    3.1105    1.0000    5232.6     0.3825E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.256    3.1120    1.0000    5232.6     0.3665E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 26.089    3.1140    1.0000    5232.6     0.3502E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 90.000    3.6151    1.0000    5232.6     0.3477E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 85.000    3.6568    1.0000    5232.6     0.4780E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 80.000    3.6968    1.0000    5232.6     0.6420E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 75.000    3.7332    1.0000    5232.6     0.8338E-06   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 70.000    3.7635    1.0000    5232.6     0.1036E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 65.000    3.7837    1.0000    5232.6     0.1212E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 60.000    3.7884    1.0000    5232.6     0.1305E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 55.000    3.7709    1.0000    5232.6     0.1315E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 50.000    3.7227    1.0000    5232.6     0.1609E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 48.000    3.6928    1.0000    5232.6     0.2026E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 46.000    3.6562    1.0000    5232.6     0.2729E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 44.000    3.6123    1.0000    5232.6     0.3741E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 42.000    3.5611    1.0000    5232.6     0.4909E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 40.000    3.5027    1.0000    5232.6     0.5888E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 38.000    3.4378    1.0000    5232.6     0.6347E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 36.000    3.3677    1.0000    5232.6     0.6420E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 34.000    3.2949    1.0000    5232.6     0.7081E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 32.000    3.2235    1.0000    5232.6     0.8646E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 30.000    3.1604    1.0000    5232.6     0.8287E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 29.000    3.1353    1.0000    5232.6     0.6950E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 28.000    3.1172    1.0000    5232.6     0.5493E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
MFT96 R U T -1 27.000    3.1091    1.0000    5232.6     0.4366E-05   55.204899  163.244598   46.770000   85.767998 0  1 COMMENT
