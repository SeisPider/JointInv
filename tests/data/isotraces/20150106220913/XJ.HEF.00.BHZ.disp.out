SURF96 R U T -1 90.000    3.6151    1.0000    
SURF96 R C T -1 90.000    6.4657    1.0000    
SURF96 R U T -1 85.000    3.6568    1.0000    
SURF96 R C T -1 85.000    6.1975    1.0000    
SURF96 R U T -1 80.000    3.6968    1.0000    
SURF96 R C T -1 80.000    5.9573    1.0000    
SURF96 R U T -1 75.000    3.7332    1.0000    
SURF96 R C T -1 75.000    5.7409    1.0000    
SURF96 R U T -1 70.000    3.7635    1.0000    
SURF96 R C T -1 70.000    5.5445    1.0000    
SURF96 R U T -1 65.000    3.7837    1.0000    
SURF96 R C T -1 65.000    5.3648    1.0000    
SURF96 R U T -1 60.000    3.7884    1.0000    
SURF96 R C T -1 60.000    5.1983    1.0000    
SURF96 R U T -1 55.000    3.7709    1.0000    
SURF96 R C T -1 55.000    5.0409    1.0000    
SURF96 R U T -1 50.000    3.7227    1.0000    
SURF96 R C T -1 50.000    4.8878    1.0000    
SURF96 R U T -1 48.000    3.6928    1.0000    
SURF96 R C T -1 48.000    4.8264    1.0000    
SURF96 R U T -1 46.000    3.6562    1.0000    
SURF96 R C T -1 46.000    4.7642    1.0000    
SURF96 R U T -1 44.000    3.6123    1.0000    
SURF96 R C T -1 44.000    4.7007    1.0000    
SURF96 R U T -1 42.000    3.5611    1.0000    
SURF96 R C T -1 42.000    4.6352    1.0000    
SURF96 R U T -1 40.000    3.5027    1.0000    
SURF96 R C T -1 40.000    4.5673    1.0000    
SURF96 R U T -1 38.000    3.4378    1.0000    
SURF96 R C T -1 38.000    4.4962    1.0000    
SURF96 R U T -1 36.000    3.3677    1.0000    
SURF96 R C T -1 36.000    4.4214    1.0000    
SURF96 R U T -1 34.000    3.2949    1.0000    
SURF96 R C T -1 34.000    4.3424    1.0000    
SURF96 R U T -1 32.000    3.2235    1.0000    
SURF96 R C T -1 32.000    4.2590    1.0000    
SURF96 R U T -1 30.000    3.1604    1.0000    
SURF96 R C T -1 30.000    4.1716    1.0000    
SURF96 R U T -1 29.000    3.1353    1.0000    
SURF96 R C T -1 29.000    4.1269    1.0000    
SURF96 R U T -1 28.000    3.1172    1.0000    
SURF96 R C T -1 28.000    4.0818    1.0000    
SURF96 R U T -1 27.000    3.1091    1.0000    
SURF96 R C T -1 27.000    4.0368    1.0000    
